index.html

```diff
@@ -54,9 +54,14 @@
 
 <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.4/handlebars.js"></script>
+<script src="/vendor/page.js"></script>
 <script src="js/app.js"></script>
+<script src="scripts/aboutController.js"></script>
+<script src="scripts/homeController.js"></script>
+<script src="scripts/projectsController.js"></script>
```

Comment:
I have seen problems with page.js arising from not having a / in front of your pathes. That might be part of your problem?